CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.4.2-dev (unreleased)
```

Comment:
we don't release dev versions yet so let's stop with `-dev` suffix, just add version and keep it unreleased.